command/pr_create.go

```diff
@@ -68,14 +101,28 @@ func prCreate(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("could not parse body: %w", err)
 	}
 
+	defs, err := computeDefaults(baseBranch, headBranch)
+	if err != nil {
+		fmt.Fprintf(colorableErr(cmd), "%s warning: could not compute title or body defaults:  %w\n", utils.Yellow("!"), err)
```

Comment:
Just had a thought: if `--title` and `--body` were explicitly passed, then we don't ever have to show this warning message because it doesn't matter whether looking up `git log` succeeded or failed.

Maybe we can save the error message in a specific variable and later:
1. Show it as a warning if explicit title & body weren't passed;
2. Show it as a fatal error if `--fill` was requested.

Does this make sense?

Reply:
yup! i like that approach and will do it